Assets/Scripts/PlayerInteraction/HeadsetCollide.cs

```diff
@@ -6,38 +6,27 @@
 
 namespace Anivision.PlayerInteraction
 {
-    public class HeadsetCollide : MonoBehaviour
+    /// <summary>
+    /// Script attached to the center eye to detect collisions with other objects.
+    /// Invokes a UnityEvent with a parameter of the colliding gameobject's Collider.
+    /// </summary>
+    [System.Serializable]
+    public class OnCollideEvent : UnityEvent<Collider>
     {
-        private Grabber[] _Grabbers;
-        private Grabber _RGrabber;
-        private Grabber _LGrabber;
+    }
```

Comment:
this doesn't need to be serialized if you're not showing in inspector, and it can be moved inside the HeadsetCollide class itself